### ISSUE TITLE
Enhancement idea

### ISSUE THREAD

**Issue by Jahfry** [Rexeh/joystick-diagrams#65]
3 things I'd enjoy seeing in a future version, no rush, I realize both of these are a bit beyond current scope. And some of this might be doable now, I just didn't find it with a cursory use of the app tonight. 

1) Ability to display 2 devices on a single diagram (or more than 2) 

2) Dark mode ... basically just a way to switch text output on the diagrams to white or a different bright color as the rest of this can be done by editing the SVGs. 

3) Ability to see button/axis usage on a "live" diagram, as in not in an SVG but using the SVG as the background for a live application that displays stick usage.

Combining the 3 above would make this a really nice visualizer for videos & streaming.

**Comment by Rexeh** [comment]
Thanks for these @Jahfry 

Can I get some context for item 3? How would this be used, be interesting to see the case!

**Comment by Jahfry** [comment]
Exactly. I've used a couple of OBS plugins in the past that could display input from sticks but setup was a chore. Being able to create an SVG with a program that understands the inputs better would be useful. 

That's also the driver behind the dark mode text request. Maybe based on values in the SVG. Like a text string called "textcolor" that is stripped from the output but text in the output takes whatever color was assigned to that string (if this doesn't make sense I could mock it up). That way the text in the "live diagram" could be stream friendly. 

The end result being a steamer could create a SVG that works well with keying out the background, and use the output as a video source in OBS. 

Way out of current scope of your application for sure, but an idea for future directions :)
